com.heroicgameslauncher.hgl.yml

```diff
@@ -80,16 +80,16 @@ add-extensions:
 modules:
 
   # --- Tools ---
-  - name: unappimage
-    buildsystem: simple
-    build-commands:
-      - make -C squashfs-tools -j ${FLATPAK_BUILDER_N_JOBS} install INSTALL_DIR=/app/bin
-    cleanup: ["*"]
-    sources:
-      - type: git
-        url: https://github.com/refi64/unappimage
-        commit: d7f86f2a0d7ec3a69211125207d5f127386b849a
-
+#   - name: unappimage
```

Comment:
Looks good. Can you remove the comments?

Reply:
I was planning on keeping them *just in case* they were needed in the future (such as an AppImage not working with it), but yeah, I can